tests/buildkite/pipeline-mgpu.yml

```diff
@@ -0,0 +1,29 @@
+env:
+  DOCKER_CACHE_ECR_ID: "492475357299"
+  DOCKER_CACHE_ECR_REGION: "us-west-2"
+  DISABLE_RELEASE: "1"
+    # Skip uploading artifacts to S3 bucket
+    # Also, don't build all CUDA archs; just build sm_75
+steps:
+  - block: ":rocket: Run this test job"
+    if: build.pull_request.repository.fork == true
```

Comment:
Note: for now, multi-GPU tests will always run for every commit of the `master` branch.